G/GDAL/build_tarballs.jl

```diff
@@ -200,30 +195,30 @@ hdf5_platforms = expand_cxxstring_abis(hdf5_platforms)
 # Dependencies that must be installed before this package can be built
 dependencies = [
     BuildDependency(PackageSpec(; name="OpenMPI_jll", version=v"4.1.6"); platforms=filter(p -> nbits(p)==32, platforms)),
-    Dependency("Arrow_jll"; compat="18.1.0"),
-    Dependency("Blosc_jll"; compat="1.21.1"),
-    Dependency("Expat_jll"; compat="2.2.10"),
-    Dependency("GEOS_jll"; compat="3.11.2"),
+    Dependency("Arrow_jll"; compat="19.0.0"),
+    Dependency("Blosc_jll"; compat="1.21.6"),
+    Dependency("Expat_jll"; compat="2.6.5"),
+    Dependency("GEOS_jll"; compat="3.13.1"),
```

Comment:
GEOS devs believe it's a good idea to break the ABI in every single version: https://github.com/JuliaBinaryWrappers/GEOS_jll.jl/commit/e2e83661fbd85faedbd29485ac0a328338321f83#diff-0ed6f04e79778dda22f5565f89d1f6e95329a8d15c57c4efa586fbe7b7374bcd
```suggestion
    Dependency("GEOS_jll"; compat="=3.13.1"),
```
But also, hold on this on https://github.com/JuliaRegistries/General/pull/126663